text/metrics/0128-instrument-multi-aggregators.md

```diff
@@ -0,0 +1,82 @@
+# SDK Proposal: Default and Configurable Aggregations for Instruments
+
+This proposal builds off the work of John in
+[OTEP-126](https://github.com/open-telemetry/oteps/pull/126) and Chris in
+[OTEP-89](https://github.com/open-telemetry/oteps/pull/89).
+
+The unique part of this proposal is that all metric exports are based on
+views. Each `Instrument` has a default `View` created when the `Instrument` is
+created.
+
+## Definitions
+
+* Measurement: The value recorded along with a set of label keys and values.
+* Instrument: The thing a `Measurement` is associated with when recorded. Each
+  has a name, description, instrument kind and unit.
+* Aggregation: Defines how a set of measurements for an instrument are to be
+  combined for export.
+* View: An aggregation on an `Instrument` and some metadata (a name,
+  description, etc).
+
+## View
+
+* name: The name used when exporting metrics. 
+* description: String value describing what the aggregated measurements represent.
+* aggregation: The implementation of combining measurements to use.
+* labels: The labels to filter down to from the labels included in the measurement.
+* exporter: Each `view` can be associated with an exporter. By default it is set
```

Comment:
Do we need this now? This can be done in the exporter itself if needed. Trying to limit the work needed initially.

Reply:
Could be. There will be far more views than exporters, and likely usually 1, so N views to 1 exporter, making configuration in the exporter be based on saying what of the N views to export while configuration on the view is a limited set of exporters to pick from. Just seems easier to put configuration on the thing that has fewer options to choose from.